scripts/aggregate_cells.py

```diff
@@ -0,0 +1,67 @@
+import os
+import faulthandler
+import pandas as pd
+from pycytominer.cyto_utils.cells import SingleCells
+
+
+def aggregate(sql_file, metadata_dir, barcode_platemap):
+    """_summary_
```

Comment:
Is this for some autodoc package?

Reply:
Nope, my editor automatically generates documentation strings when ever creating a function. I forgot to complete the documentation. 

changes applied on commit: 21305ee51b64193213e31d112de1f6e5e17eff03